test/e2e/dra/deploy.go

```diff
@@ -136,6 +136,7 @@ func (d *Driver) SetUp(nodes *Nodes, resources app.Resources) {
 	ginkgo.By(fmt.Sprintf("deploying driver on nodes %v", nodes.NodeNames))
 	d.Nodes = map[string]*app.ExamplePlugin{}
 	d.Name = d.f.UniqueName + d.NameSuffix + ".k8s.io"
+	resources.DriverName = d.Name
```

Comment:
This modifies the local copy of the `resources` variable. Is the expectation is there any expectation that the `DriverName` be queried after `SetUp` is called?

Would it be cleaner to call `configureResources` from `SetUp` instead and not pass in `resources`?

Reply:
This code gets called by E2E tests which only have `app.Resources` in memory (i.e. no config file), so it has to be a parameter. None of those tests currently set the driver name in `resources` and instead expect the name to be generated. This code basically keeps that behavior.